packages/astro/src/client/index.ts

```diff
@@ -14,7 +17,10 @@ export const createClerkInstance = runOnce(createClerkInstanceInternal);
 export async function createClerkInstanceInternal(options?: AstroClerkIntegrationParams) {
   let clerkJSInstance = window.Clerk;
   if (!clerkJSInstance) {
-    await waitForClerkScript();
+    await loadClerkJsScript({
+      clerkJSVersion: HARDCODED_LATEST_CLERK_JS_VERSION,
```

Comment:
We'll need to remove this, I'll explain it in https://github.com/clerk/javascript/pull/3681/files#r1672868119